source.js

```diff
@@ -32,6 +32,7 @@ function buildSourceMetadata(source, context) {
         user: context.actor,
         name: context.eventName,
         workflow: context.workflow,
+        url: `https://github.com/${full_name}/actions/runs/${context.runId}/attempts/${process.env.GITHUB_RUN_ATTEMPT || "1"}`
```

Comment:
Oh nice we get the run attempt for free!

Double checked this for a github action that without no re-runs, and looks like both work just fine:
- https://github.com/jaril/test-pw/actions/runs/2452372958
- https://github.com/jaril/test-pw/actions/runs/2452372958/attempts/1

Reply:
Yeah. I checked that too.